cypress/pages/HeaderNavigationElements.page.js

```diff
@@ -0,0 +1,15 @@
+export class HeaderNavigationElements {
+
+    elements = {
+      logo: () => cy.get('.logo'),
+      userIcon: () => cy.get('.user-picture'),
+      light: () => cy.get('button.select-button'),
```

Comment:
```suggestion
      lightButton: () => cy.get('button.select-button'),
```
я б вказував конкретно який це елемент